platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/reconnect/ReconnectLearner.java

```diff
@@ -192,7 +192,12 @@ private ReservedSignedState reconnect() throws InterruptedException {
 
         final State state = (State) synchronizer.getRoot();
         final SignedState newSignedState = new SignedState(
-                platformContext, CryptoStatic::verifySignature, state, "ReconnectLearner.reconnect()", false);
+                platformContext,
+                CryptoStatic::verifySignature,
+                state,
+                "ReconnectLearner.reconnect()",
+                false,
+                true); // TODO will this cause things to be deleted incorrectly during tests?
```

Comment:
TODO

Reply:
I traced the code pathways. There are no virtual map tests that execute this code, so it is safe to leave as `true`.